decidim-admin/spec/features/manage_participatory_processes_spec.rb

```diff
@@ -37,12 +37,12 @@
       expect(page).to have_content(participatory_process.subtitle["en"])
       expect(page).to have_content(participatory_process.subtitle["es"])
       expect(page).to have_content(participatory_process.subtitle["ca"])
-      expect(page).to have_content(participatory_process.short_description["en"])
-      expect(page).to have_content(participatory_process.short_description["es"])
-      expect(page).to have_content(participatory_process.short_description["ca"])
-      expect(page).to have_content(participatory_process.description["en"])
-      expect(page).to have_content(participatory_process.description["es"])
-      expect(page).to have_content(participatory_process.description["ca"])
+      expect(page).to have_content(participatory_process.short_description["en"].gsub(/<\/?[^>]*>/, ""))
```

Comment:
Either way, I think you're overtesting. Acceptance testing should only care about the most valuable stuff a particular feature is about. Here, it looks like you're actually testing `display_attribute` instead. This makes the test fragile as you're coupling it with this particular replacement (`gsub`) a lot of times and with the available locales as well.